README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img src="images/README/logo.png" style="width:300px;" />
+</p>
+
 Grofer
```

Comment:
Can we change title to `grofer`? idk capital G looks weird :P

Reply:
Fair enough. Will change,